Makefile

```diff
@@ -170,15 +172,18 @@ cluster/cleanup/crds:
 	@-oc delete crd installations.integreatly.org
 	@-oc delete crd webapps.integreatly.org
 
+.PHONY: deploy/integreatly-installation-cr.yml
```

Comment:
I'm not sure about the need for this to be .PHONY -- it's a target that corresponds to a file that is created as a result of running it, so it's a more normal target than our usual ones, right?

If we're going to change it to a phony target, it might be worth changing the name to something that's not also a file path, e.g. `deploy/cr`.

Oh gosh, now I'm questioning the sense in having phony targets that look like file paths (slashy) :thinking: